charts/AliceMicroservices/README.md

```diff
@@ -0,0 +1,6 @@
+#Plan - Helm chart
```

Comment:
Same here, please change the description and add what all is supported in this current chart

Reply:
made changes as needed @pankajnarang